lib/vanagon/platform/rpm.rb

```diff
@@ -45,15 +45,15 @@ def package_name(project)
         "#{project.name}-#{project.version}-#{project.release}.#{project.noarch ? 'noarch' : @architecture}.rpm"
       end
 
-      def output_dir(target_repo = "products")
-        super
-      end
-
       # Method to derive the directory for source artifacts
       #
       # @param target_repo [String] repo the source artifacts are targeting
-      def source_output_dir(target_repo = "products")
-        @source_output_dir ||= File.join(@os_name, @os_version, target_repo, 'SRPMS')
+      def source_output_dir(target_repo = nil)
+        if target_repo.nil?
+          super
```

Comment:
what do these end up looking like when there's no target_repo? are we going to end up with the srpm/source tags like http://enterprise.delivery.puppetlabs.net/2017.3/repos/ ?